portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,7 +63,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
     String comment = getParameter(request, "text-input", "");
-    comments.add(comment);
+    // comments.add(comment);
+    long timestamp = System.currentTimeMillis();
+
+    Entity commentEntity = new Entity("Comment");
```

Comment:
use static final string variable for "Comment"

Reply:
done.